src/evict/evict_page.c

```diff
@@ -179,9 +179,14 @@ __evict_delete_ref(WT_SESSION_IMPL *session, WT_REF *ref, bool closing)
 		 * something is busy, be sure that the page still ends up
 		 * marked deleted.
 		 */
-		if (ndeleted > pindex->entries / 10 && pindex->entries > 1 &&
-		    (ret = __wt_split_reverse(session, ref)) != EBUSY)
-			return (ret);
+		if (ndeleted > pindex->entries / 10 && pindex->entries > 1) {
+			if ((ret = __wt_split_reverse(session, ref)) == 0)
+				return (ret);
+			WT_RET_BUSY_OK(ret);
+
+			/* If a reverse split fails, unlock the child. */
+			WT_ASSERT(session, ref->state == WT_REF_LOCKED);
```

Comment:
This code path is now eating the error return from ``__wt_split_reverse`` - I think the return statement should be updated to ``return (ret)``